Modules/Loadable/Markups/MRML/vtkMRMLMarkupsPlaneNode.cxx

```diff
@@ -951,6 +951,58 @@ void vtkMRMLMarkupsPlaneNode::SetSize(double x, double y)
   this->Modified();
 }
 
+//----------------------------------------------------------------------------
+void vtkMRMLMarkupsPlaneNode::GetSizeWorld(double size_World[2])
+{
+  vtkNew<vtkMatrix4x4> objectToWorldMatrix;
+  this->GetObjectToWorldMatrix(objectToWorldMatrix);
+  vtkNew<vtkTransform> objectToWorldTransform;
+  objectToWorldTransform->SetMatrix(objectToWorldMatrix);
+
+  double sizeX_Object3[3] = { this->Size[0], 0.0, 0.0 };
```

Comment:
This `sizeX_Object3` name is not clear. Could you rename it to something like `scaledAxisXVector_Object` or `sideXVector_Object` or something similar(include `Vector` in the name and don't call it size, because you already use size for scalars).